pom.xml

```
@@ -685,26 +706,11 @@
                 <groupId>org.eclipse.sisu</groupId>
                 <artifactId>org.eclipse.sisu.plexus</artifactId>
                 <version>0.3.4</version>
-            </dependency>
-            <dependency>
-                <groupId>org.opensearch</groupId>
-                <artifactId>opensearch</artifactId>
-                <version>1.2.3</version>
-            </dependency>
-            <dependency>
-                <groupId>org.opensearch</groupId>
-                <artifactId>opensearch-core</artifactId>
-                <version>1.2.3</version>
-            </dependency>
-            <dependency>
-                <groupId>org.opensearch</groupId>
-                <artifactId>opensearch-x-content</artifactId>
-                <version>1.2.3</version>
-            </dependency>
+            </dependency> 
```

Extra space on line:

```suggestion
            </dependency>
```